src/components/BookForm/BookForm.css

```diff
@@ -19,9 +19,15 @@
 
 .bookForm h2 {
   margin-top: 0;
+  margin-bottom: 0;
 }
 
-form input {
+form {
+  padding: 15px 20px 0;
+}
+
+form input,
+select {
```

Comment:
- [ ] It is a requirement that your design matches the given design kindly do so with your design.

Your design
![image](https://user-images.githubusercontent.com/81932478/207863186-f2475cb6-9175-4d46-a394-fdeb2f25c606.png)

Given design
![image](https://user-images.githubusercontent.com/81932478/207863285-e13eff09-32bd-4f76-a638-b03637943441.png)

#### note thta this includes color schemes, icons typography.